LabExT/Experiments/StandardExperiment.py

```diff
@@ -409,6 +409,48 @@ def _write_metadata(self, target: dict = None, file_path: str = "tmp.json") -> d
 
         return target
 
+    def __guess_params(self, meas_dict: MeasurementDict) -> dict:
+        """Tries to guess the parameters used for a certain measurement based on the "measurement settings".
+
+        This method is only used for old measurements who did not specify the "measurement_params" entry.
+        To guess the parameters we use the following scheme:
+        1. Check if the entry "measurement settings" exists
+        2. Check if the entry is a dict or dict-like
+        3. Iterate through the "keys" of the dict-like object
+            a. If the corresponding value is dict-like itself, try to get the value corresponding to the key "value".
+            b. Otherwise assume the value is some sort of float/int/bool/... and use it directly
+
+        Args:
+            meas_dict: The meas_dict which is used to guess the used parameters.
+
+        Returns:
+            A new dict containing the guessed name-value pairs.
+        """
```

Comment:
This function should become superfluous once the requested changes in #199 are implemented. Please do not use two different keys to store the same measurement parameters.

Reply:
That's correct. Removed in 73b76d749fac742ecbb6c31cd6045c68d7107887.